index.bs

```diff
@@ -7052,6 +7052,15 @@ the [=[RP]=] could mitigate the privacy leak using the same approach of returnin
 as discussed in [[#sctn-username-enumeration]].
 
 
+# Accessibility Considerations # {#sctn-accessiblility-considerations}
+
+[=User verification=]-capable [=authenticators=], whether [=roaming authenticators|roaming=] or [=platform authenticators|platform=], should offer users more than one user verification method.  For example, both fingerprint sensing and PIN entry. This allows for fallback to other user verification means if the selected one is not working for some reason. Note that in the case of [=roaming authenticators=], the authenticator and platform might work together to provide a user verification method such as PIN entry [[FIDO-CTAP]].
```

Comment:
Should the "should"s throughout the section be "SHOULD"s? I see there's a normative "MAY" in the last sentence.

Reply:
I think not in the 1st parag since that is with regards to authenticators themselves and this is not an authenticator-oriented spec.

I think yes in 2nd parag, since that is wrt Relying Parties.